projects/driver_controls/inc/can_output.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+// Module for broadcasting CAN messages
+#include <stdint.h>
+#include "status.h"
+#include "event_arbiter.h"
+
+// Device ID definitions
+typedef enum {
+  CAN_OUTPUT_DEVICE_POWER = 0,
```

Comment:
Why are these still called devices? Normally, when we refer to CAN devices, we're talking about devices on the bus. These are more like messages or events.

Reply:
Done